src/utils/cookies/userInfo.js

```diff
@@ -26,9 +26,19 @@ function setUserSession(val) {
   return Cookies.set("sessionID", val);
 }
 
+function saveClickedVideoCookie() {
```

Comment:
cookies are always sent to the server and retrieved back; it seems a bit wasteful. maybe use LocalStorage? Otherwise good idea!